e2e/test/config/TestConfig.cs

```diff
@@ -5,7 +5,7 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 //Workers = 0 makes the test engine use one worker per available core. It does not mean to run serially.
-[assembly: Parallelize(Workers = 0, Scope = ExecutionScope.ClassLevel)]
+//[assembly: Parallelize(Workers = 0, Scope = ExecutionScope.ClassLevel)]
```

Comment:
This is commented out right now, to leverage parallelization at the base test class level. If it works as expected, the goal is to remove this class altogether.